plugins/i3s/io/EsriReceiver.hpp

```diff
@@ -0,0 +1,102 @@
+/******************************************************************************
```

Comment:
I don't think that EsriReceiver is a clear description of the contents of this file.  EsriUtils?  EsriSupport?